JS Tasks/Task 7/script.js

```diff
@@ -10,3 +10,19 @@ const audi = {
   year: 2005,
   color: 'white',
 };
+
+//Jei reikia gauti "key" reikšmes:
+function showObjectKeys (anyObject) {
+  const newArray = Object.values(anyObject);
+  return newArray 
+};
```

Comment:
Vertinu šitą funkciją, nes atitinka pavadinimas. Deja, reikėjo naudoti `Object.keys`.